todoman/model.py

```diff
@@ -587,6 +589,8 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             params.extend(lists)
         if urgent:
             extra_where.append('AND priority = 9')
+        elif priority:
+            extra_where.append('AND priority = ' + priority)
```

Comment:
I think it's better to show the tasks with priority *at least* as high as the given one, not the ones with the exact given priority.